src/System.Private.ServiceModel/src/System/ServiceModel/Security/SecurityUtils.cs

```diff
@@ -578,6 +660,20 @@ private static NetworkCredential UnsafeGetDefaultNetworkCredentials()
             {
                 return CredentialCache.DefaultNetworkCredentials;
             }
+
+            internal static string GetCurrentUserIdAsString(NetworkCredential credential)
+            {
+#if FEATURE_NETNATIVE
+                // There's no way to retrieve the current logged in user Id in UWP apps
+                // so returning a username which is very unlikely to be a real username;
+                return "_______****currentUser****_______";
```

Comment:
I looked into that, but unfortunately that's no longer supported in Win10. As per the [documentation](https://msdn.microsoft.com/en-us/library/windows/apps/windows.system.userprofile.userinformation.aspx) for that api:  
The **UserInformation** class is not supported on Windows 10 or later. Use the [**User**](https://msdn.microsoft.com/en-us/library/windows/apps/windows.system.user.aspx) class instead  
Unfortunately, I wasn't able to find a way to get the User object for the currently logged in user. If you know the Win10 equivalent, I'd happily put it in there.